src/main/java/org/influxdb/dto/BatchPoints.java

```diff
@@ -83,6 +83,17 @@ public Builder tag(final String tagName, final String value) {
       return this;
     }
 
+    /**
+     * Add a set of tags to this set of tags.
+     *
+     * @param tagsToAdd the list of tags to add
+     * @return the Builder instance
+     */
+    public Builder tags(final Map<String, String> tagsToAdd) {
+      this.tags.putAll(tagsToAdd);
```

Comment:
Nothing big here but if the parameter is tags and not tagsToAdd then I would use this.tags.  Otherwise, it is known to be this.tags.